src/ts_catalog/continuous_agg.c

```diff
@@ -1610,9 +1610,7 @@ ts_continuous_agg_watermark(PG_FUNCTION_ARGS)
 	AclResult aclresult;
 
 	if (PG_ARGISNULL(0))
-		ereport(ERROR,
-				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
-				 errmsg("materialized hypertable cannot be NULL")));
+		PG_RETURN_NULL();
```

Comment:
So it always returns null for null argument, this means that it's strict, but I see that you removed the STRICT attribute above?

Reply:
Wondering the same.

Reply:
Fixed!